packages/repository/test/acceptance/has-one.relation.acceptance.ts

```diff
@@ -44,25 +44,27 @@ describe('hasOne relation', () => {
       street: '123 test avenue',
     });
 
-    const persisted = await addressRepo.findById(address.customerId);
+    const persisted = await addressRepo.findById(address.zipcode);
     expect(persisted.toObject()).to.deepEqual(address.toObject());
   });
 
   it('refuses to create related model instance twice', async () => {
     const address = await controller.createCustomerAddress(existingCustomerId, {
       street: '123 test avenue',
     });
+    // FIXME(b-admike): memory connector doesn't support unique indexes
```

Comment:
Please check other commonly used connectors too! For example:
 - MongoDB
 - CouchDB/Cloudant
 - MySQL
 - PostgreSQL
 - Oracle
 - MSSQL

Reply:
So far from what I checked, MongoDB, MysQL, PostgreSQL, and Oracle support unique indexes. Will check the others and report back 👍